<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt003</class>
 <widget class="QDialog" name="qt003">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>词云02</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>文件路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="t_file">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_select_file">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>内容选择</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;屏蔽内容&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;的文件路径&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select_file_probit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_file_probit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="t_file_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>背景文件路径：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_select_file_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_show_pic">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>图片预览</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>501</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string>结果</string>
   </property>
   <widget class="QLabel" name="lbl_pic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_aim_file">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>词云范围</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rb_all">
        <property name="text">
         <string>全篇文字</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_top">
        <property name="text">
         <string>Top 50 高频词</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_cloud">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>生成词云</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;生成的词云&lt;/p&gt;&lt;p&gt;图片位置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>681</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pic_orig">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>t_file</zorder>
  <zorder>btn_select_file</zorder>
  <zorder>t_file_2</zorder>
  <zorder>label_2</zorder>
  <zorder>btn_select_file_2</zorder>
  <zorder>btn_show_pic</zorder>
  <zorder>lbl_pic_orig</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
